<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakePlaylist</class>
 <widget class="QMainWindow" name="MakePlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать плейлист</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_file">
        <property name="text">
         <string>Добавит Файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="del_file">
        <property name="text">
         <string>Удалить Файл</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="name_playlist_text">
        <property name="text">
         <string>Имя Плейлиста</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="playlist_name"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="info">
      <property name="text">
       <string>Чтобы удалить файл надо: дважды нажать на файл в списке и нажать кнопку 'Удалить Файл'</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="files"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="make_playlist">
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
